<ui version="4.0" >
 <class>UISettingsQMake</class>
 <widget class="QWidget" name="UISettingsQMake" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Settings...</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>5</number>
   </property>
   <property name="spacing" >
    <number>3</number>
   </property>
   <item>
    <widget class="QTabWidget" name="twQMake" >
     <widget class="QWidget" name="wGeneral" >
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>5</number>
       </property>
       <property name="spacing" >
        <number>5</number>
       </property>
       <item row="3" column="0" colspan="2" >
        <widget class="QGroupBox" name="gbFilter" >
         <property name="title" >
          <string>Accepted Variable Filter</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>5</number>
          </property>
          <property name="spacing" >
           <number>3</number>
          </property>
          <item row="0" column="0" >
           <widget class="QToolButton" name="tbAddFilter" >
            <property name="text" >
             <string>Add</string>
            </property>
            <property name="icon" >
             <iconset resource="../../../../../monkey/src/resources/resources.qrc" >:/Icons/Icons/editadd.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="4" >
           <widget class="QListWidget" name="lwFilters" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>13</hsizetype>
              <vsizetype>7</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="editTriggers" >
             <set>QAbstractItemView::CurrentChanged|QAbstractItemView::NoEditTriggers|QAbstractItemView::SelectedClicked</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QToolButton" name="tbRemoveFilter" >
            <property name="text" >
             <string>Remove</string>
            </property>
            <property name="icon" >
             <iconset resource="../../../../../monkey/src/resources/resources.qrc" >:/Icons/Icons/editremove.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QToolButton" name="tbClearFilters" >
            <property name="text" >
             <string>Clear</string>
            </property>
            <property name="icon" >
             <iconset resource="../../../../../monkey/src/resources/resources.qrc" >:/Icons/Icons/editclear.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="3" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" colspan="2" >
        <widget class="QComboBox" name="cbKeywords" />
       </item>
       <item row="1" column="0" >
        <widget class="QLineEdit" name="lePath" >
         <property name="toolTip" >
          <string>Enter the path or file path of the current keywords</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QToolButton" name="tbBrowse" >
         <property name="cursor" >
          <cursor>13</cursor>
         </property>
         <property name="toolTip" >
          <string>Locate path or file</string>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="../../../../../monkey/src/resources/resources.qrc" >:/Icons/Icons/fileopen.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2" >
        <widget class="QGroupBox" name="gbApi" >
         <property name="title" >
          <string>Generate Qt Api files</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>5</number>
          </property>
          <property name="spacing" >
           <number>3</number>
          </property>
          <item row="0" column="0" >
           <widget class="QCheckBox" name="cbQt4" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Qt4</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2" >
           <widget class="QProgressBar" name="pbProgress" >
            <property name="maximum" >
             <number>0</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QCheckBox" name="cbQt3" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Qt3</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QPushButton" name="pbGenerate" >
            <property name="text" >
             <string>Generate</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2" >
           <widget class="QLabel" name="lInformations" >
            <property name="frameShape" >
             <enum>QFrame::StyledPanel</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="wQtModules" >
      <attribute name="title" >
       <string>Qt Modules</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>5</number>
       </property>
       <property name="spacing" >
        <number>5</number>
       </property>
       <item row="7" column="1" >
        <widget class="QTextEdit" name="teHelpQtModule" >
         <property name="tabChangesFocus" >
          <bool>true</bool>
         </property>
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="1" >
        <widget class="QLabel" name="lHelpQtModule" >
         <property name="text" >
          <string>&amp;Help :</string>
         </property>
         <property name="buddy" >
          <cstring>teHelpQtModule</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1" >
        <widget class="QLineEdit" name="leVariableQtModule" />
       </item>
       <item row="3" column="1" >
        <widget class="QLineEdit" name="leValueQtModule" />
       </item>
       <item row="2" column="1" >
        <widget class="QLabel" name="lValueQtModule" >
         <property name="text" >
          <string>&amp;Value :</string>
         </property>
         <property name="buddy" >
          <cstring>leValueQtModule</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="QLabel" name="lVariableQtModule" >
         <property name="text" >
          <string>V&amp;ariable :</string>
         </property>
         <property name="buddy" >
          <cstring>leVariableQtModule</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLabel" name="lCaptionQtModule" >
         <property name="text" >
          <string>&amp;Caption :</string>
         </property>
         <property name="buddy" >
          <cstring>leCaptionQtModule</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="leCaptionQtModule" />
       </item>
       <item rowspan="8" row="0" column="0" >
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>3</number>
         </property>
         <item row="0" column="2" >
          <widget class="QToolButton" name="tbClearQtModules" >
           <property name="text" >
            <string>Clear</string>
           </property>
           <property name="icon" >
            <iconset resource="../../../../../monkey/src/resources/resources.qrc" >:/Icons/Icons/editclear.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="3" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1" >
          <widget class="QToolButton" name="tbRemoveQtModule" >
           <property name="text" >
            <string>Remove</string>
           </property>
           <property name="icon" >
            <iconset resource="../../../../../monkey/src/resources/resources.qrc" >:/Icons/Icons/editremove.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="4" >
          <widget class="QListWidget" name="lwQtModules" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>13</hsizetype>
             <vsizetype>7</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editTriggers" >
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QToolButton" name="tbAddQtModule" >
           <property name="text" >
            <string>Add</string>
           </property>
           <property name="icon" >
            <iconset resource="../../../../../monkey/src/resources/resources.qrc" >:/Icons/Icons/editadd.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="wSettings" >
      <attribute name="title" >
       <string>Settings</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>5</number>
       </property>
       <property name="spacing" >
        <number>5</number>
       </property>
       <item row="0" column="1" >
        <widget class="QLabel" name="lCaptionSetting" >
         <property name="text" >
          <string>&amp;Caption :</string>
         </property>
         <property name="buddy" >
          <cstring>leCaptionSetting</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="leCaptionSetting" />
       </item>
       <item row="2" column="1" >
        <widget class="QLabel" name="lValueSetting" >
         <property name="text" >
          <string>&amp;Value :</string>
         </property>
         <property name="buddy" >
          <cstring>leValueSetting</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QLineEdit" name="leValueSetting" />
       </item>
       <item row="4" column="1" >
        <widget class="QLabel" name="lVariableSetting" >
         <property name="text" >
          <string>V&amp;ariable :</string>
         </property>
         <property name="buddy" >
          <cstring>leVariableSetting</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1" >
        <widget class="QLineEdit" name="leVariableSetting" />
       </item>
       <item row="6" column="1" >
        <widget class="QLabel" name="lHelpSetting" >
         <property name="text" >
          <string>&amp;Help :</string>
         </property>
         <property name="buddy" >
          <cstring>teHelpSetting</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="1" >
        <widget class="QTextEdit" name="teHelpSetting" >
         <property name="tabChangesFocus" >
          <bool>true</bool>
         </property>
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item rowspan="8" row="0" column="0" >
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>3</number>
         </property>
         <item row="0" column="2" >
          <widget class="QToolButton" name="tbClearSettings" >
           <property name="text" >
            <string>Clear</string>
           </property>
           <property name="icon" >
            <iconset resource="../../../../../monkey/src/resources/resources.qrc" >:/Icons/Icons/editclear.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="3" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1" >
          <widget class="QToolButton" name="tbRemoveSetting" >
           <property name="text" >
            <string>Remove</string>
           </property>
           <property name="icon" >
            <iconset resource="../../../../../monkey/src/resources/resources.qrc" >:/Icons/Icons/editremove.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="4" >
          <widget class="QListWidget" name="lwSettings" >
           <property name="editTriggers" >
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QToolButton" name="tbAddSetting" >
           <property name="text" >
            <string>Add</string>
           </property>
           <property name="icon" >
            <iconset resource="../../../../../monkey/src/resources/resources.qrc" >:/Icons/Icons/editadd.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>301</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="bbDialog" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>4</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor" >
        <cursor>13</cursor>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Apply</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>twQMake</tabstop>
  <tabstop>cbKeywords</tabstop>
  <tabstop>lePath</tabstop>
  <tabstop>tbBrowse</tabstop>
  <tabstop>cbQt4</tabstop>
  <tabstop>cbQt3</tabstop>
  <tabstop>pbGenerate</tabstop>
  <tabstop>tbAddQtModule</tabstop>
  <tabstop>tbRemoveQtModule</tabstop>
  <tabstop>tbClearQtModules</tabstop>
  <tabstop>lwQtModules</tabstop>
  <tabstop>leCaptionQtModule</tabstop>
  <tabstop>leValueQtModule</tabstop>
  <tabstop>leVariableQtModule</tabstop>
  <tabstop>teHelpQtModule</tabstop>
  <tabstop>tbAddSetting</tabstop>
  <tabstop>tbRemoveSetting</tabstop>
  <tabstop>tbClearSettings</tabstop>
  <tabstop>lwSettings</tabstop>
  <tabstop>leCaptionSetting</tabstop>
  <tabstop>leValueSetting</tabstop>
  <tabstop>leVariableSetting</tabstop>
  <tabstop>teHelpSetting</tabstop>
  <tabstop>bbDialog</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../../monkey/src/resources/resources.qrc" />
 </resources>
 <connections/>
</ui>
